includes/class-wc-payments-account.php

```diff
@@ -1267,10 +1277,12 @@ private function settings_changed( $changes = [] ) {
 	public function possibly_update_wcpay_account_locale( $option_name, $old_value, $new_value ) {
 		if ( 'WPLANG' === $option_name && $this->is_stripe_connected() ) {
 			try {
-				$account_settings = [
-					'locale' => $new_value ? $new_value : 'en_US',
-				];
-				$updated_account  = $this->payments_api_client->update_account( $account_settings );
+				$locale  = $new_value ?? 'en_US';
+				$request = Update_Account::create();
+				$request->set_locale( $locale );
```

Comment:
Nitpick. You don't have to define $locale variable. You can move definition as a function call argument.

Reply:
No longer relevant. Closing this.